index.html

```diff
@@ -26,50 +26,275 @@
+                  <img src="./images/Counter.png" alt="" /></span
+                >back end dev
+              </p>
+              <p class="profession">
+                <span class="prof-span">
+                  <img src="./images/Counter.png" alt="" /></span
+                >2015
+              </p>
+            </div>
+            <div class="info-2">
+              <p class="info-p">
+                A daily selection of privately <br />
+                personalized reads; no accounts or sign-ups required.
+              </p>
+            </div>
+
+            <div class="ul">
+              <ul class="tech-stack">
+                <li>HTML</li>
+                <li>CSS</li>
+                <li>Javascript</li>
+              </ul>
+            </div>
+
+            <button class="project-btn">see project</button>
+          </div>
+        </article>
+      </section>
     </section>
+
+      <section class="about-myself">
+        <article class="about-article">
+          <h1 class="about-h1">about myself</h1>
+          <p class="about-p">
+            Hello I’m a software developer! I can help you build a product ,
+            feature or website. Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don’t
+            hesitate to contact me.
+          </p>
+          <div>
+            <p class="connect-about">let's connect</p>
+            <ul class="social-icon">
+              <li>
+                <a href="https://twitter.com/toonmystory"
+                  ><img src="./images/twitter.svg" alt="" class="icon-img"
+                /></a>
+              </li>
+              <li>
+                <a href="https://www.linkedin.com/in/blessing-asuquo-4509981a2/"
+                  ><img src="./images/linkedin.png" alt="" class="icon-img"
+                /></a>
+              </li>
+              <li>
+                <a href="https://medium.com/@blessingmike334"
+                  ><img src="./images/medium.svg" alt="" class="icon-img"
+                /></a>
+              </li>
+              <li>
+                <a href="https://github.com/blessing-michael"
+                  ><img src="./images/github.svg" alt="" class="icon-img"
+                /></a>
+              </li>
+              <li>
+                <a href="https://github.com/blessing-michael"
+                  ><img src="./images/Vector (3).svg" alt="" class="icon-img"
+                /></a>
+              </li>
+            </ul>
+           
+          </div>
+         
+         
+          <div class="stack">
+            <div class="btn-container">
+                <button class="resume-btn">Get my resume</button>
+    
+              </div>
+            
+            <h4 class="about-title">language</h4>
+           
+           
+            <ul>
+              <li class="about-li">Javascript</li>
+              <li>HTML</li>
+              <li class="img-1">CSS</li>
+            </ul>
```

Comment:
- [ ] Please 🙏🏽 use the correct styles to correct the background dimension, text position, and font styles 👇🏽 

| Figma template | Your design |
|---|---|
| ![image](https://github.com/blessing-michael/portfolio/assets/58642949/7d6190e7-e592-49b6-b08b-d72af5d0458f) | ![image](https://github.com/blessing-michael/portfolio/assets/58642949/54071d0d-c2e4-4421-8449-bd5cc0799020) |